tutorials-list.json

```diff
@@ -13,6 +13,22 @@
 			"video_url": "https://github.com/user-attachments/assets/b5d6f290-8716-42de-be2b-20b1b1f0a5cb",
 			"tutorial_id": "link_to_a_new_git_repository_b5d6f290-8716-42de-be2b-20b1b1f0a5cb",
 			"component": "link_git_repository"
+		},
+		{
+			"title": "Dify for beginners: Build a RAG chatbot with LLMs",
+			"description": "Build a Dify RAG chatbot for PDF files",
+			"usage_docs_url": "https://www.notion.so/shakudo/Build-a-Dify-LLM-Chatbot-with-RAG-Beginner-32f896e698d8433480715129fad4d2ee?pvs=4",
+			"preview_image_url": "https://github.com/user-attachments/assets/8a040f72-b005-4f0a-afbe-d08cb3b2b89e",
```

Comment:
TODO: make a slightly nicer graphic with the stack components at the top of the repo